### PR TITLE
update values yaml

### PR DESCRIPTION
**Pull request by BharadwajSairam**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/PRL-5301

### Change description ###

Citizen SOA - Statement of service 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/prl/models/serviceofapplication/CitizenSos.java [hunk 1]
```diff
@@ -14,5 +13,6 @@
 public class CitizenSos {
     private final String partiesServedDate;
     private final String partiesServed;
```
Comment (shashisk97) [inline src/main/java/uk/gov/hmcts/reform/prl/models/serviceofapplication/CitizenSos.java:15]: Change it to Array of string

File: src/main/java/uk/gov/hmcts/reform/prl/models/serviceofapplication/CitizenSos.java [hunk 2]
```diff
@@ -14,5 +13,6 @@
...
-    private final List<String> citizenSosDocs;
+    private final Document citizenSosDocs;
+    private final String isOrder;
```
Comment (shashisk97) [inline src/main/java/uk/gov/hmcts/reform/prl/models/serviceofapplication/CitizenSos.java:17]: Change it to YesOrNo

File: src/main/java/uk/gov/hmcts/reform/prl/services/ServiceOfApplicationService.java [hunk 3]
```diff
@@ -1163,10 +1163,16 @@ private void sendEmailToApplicantLipPersonalC100(CaseData caseData,
...
         Map<String, Object> caseDataMap = callbackRequest.getCaseDetails().getData();
-        if (caseData.getServiceOfApplication() != null && SoaCitizenServingRespondentsEnum.unrepresentedApplicant
-            .equals(caseData.getServiceOfApplication().getSoaCitizenServingRespondentsOptionsCA())) {
-            caseData.getApplicants().get(0).getValue().getResponse().getCitizenFlags().setIsApplicationToBeServed(YesOrNo.Yes);
-            caseDataMap.put(APPLICANTS, caseData.getApplicants());
+        if (caseData.getServiceOfApplication() != null) {
+            if (SoaCitizenServingRespondentsEnum.unrepresentedApplicant
+                .equals(caseData.getServiceOfApplication().getSoaCitizenServingRespondentsOptionsCA())) {
+                caseData.getApplicants().get(0).getValue().getResponse().getCitizenFlags().setIsApplicationToBeServed(YesOrNo.Yes);
+                caseDataMap.put(APPLICANTS, caseData.getApplicants());
```
Comment (shashisk97) [inline src/main/java/uk/gov/hmcts/reform/prl/services/ServiceOfApplicationService.java:1170]: @yogendraU Can we discuss on this, so to handle the same for DA applicant LIP

File: src/main/java/uk/gov/hmcts/reform/prl/services/ServiceOfApplicationPostService.java [hunk 4]
```diff
@@ -308,7 +309,7 @@ private BulkPrintDetails sendBulkPrint(CaseData caseData, String authorisation,
         return BulkPrintDetails.builder()
             .bulkPrintId(bulkPrintedId)
             .servedParty(servedParty)
-            .printedDocs(String.join(",", docs.stream().map(Document::getDocumentFileName).toList()))
+            .printedDocs(String.join(",", docs.stream().filter(Objects::nonNull).map(Document::getDocumentFileName).toList()))
```
Comment (BharadwajSairam) [inline src/main/java/uk/gov/hmcts/reform/prl/services/ServiceOfApplicationPostService.java:312]: Validate the change!

File: src/main/java/uk/gov/hmcts/reform/prl/services/StmtOfServImplService.java [hunk 5]
```diff
@@ -342,4 +356,156 @@ private String getPartyIds(String caseTypeOfApplication,
...
+        log.info("Unserved respondent pack {}", updatedCaseDataMap.get(UN_SERVED_RESPONDENT_PACK));
+        if (null != updatedCaseDataMap.get(UN_SERVED_RESPONDENT_PACK)
+            && CollectionUtils.isNotEmpty(updatedCaseData.getServiceOfApplication().getUnServedRespondentPack().getPackDocument())) {
+            List<Element<StmtOfServiceAddRecipient>> stmtOfServiceforApplication = new ArrayList<>();
+            updateStatementOfServiceCollection(sosObject, updatedCaseData, stmtOfServiceforApplication);
+            updatedCaseDataMap.put(STMT_OF_SERVICE_FOR_APPLICATION, stmtOfServiceforApplication);
+            if (YesOrNo.Yes.getDisplayedValue().equalsIgnoreCase(sosObject.getIsOrder())) {
+                List<Element<ServedApplicationDetails>> finalServedApplicationDetailsList;
+                List<Element<BulkPrintDetails>> bulkPrintDetails = new ArrayList<>();
+                finalServedApplicationDetailsList = updateFinalListOfServedApplications(
```
Comment (BharadwajSairam) [inline src/main/java/uk/gov/hmcts/reform/prl/services/StmtOfServImplService.java:374]: AccessCode Notifications sent to respondent should be recorded as part of a collection and stored in db

Comment (BharadwajSairam) [reply src/main/java/uk/gov/hmcts/reform/prl/services/StmtOfServImplService.java:374]: Also check contact preference before sending access code letter

**Comment by BharadwajSairam** [comment]
Merge into https://github.com/hmcts/prl-cos-api/pull/2544/
Can be tested as part of that PR
So closing this

**Pull request closed without merging**